hw/ip/usbdev/data/usbdev.hjson

```diff
@@ -715,22 +715,6 @@
                 Recognize a single SE0 bit as an end of packet, otherwise two successive bits are required.
                 '''
         }
-        {
```

Comment:
btw, i will just note that the override function was sometimes useful for testing (ie, faking some kind of external behavior) :)

Reply:
There's a similar override capability in the pinmux, so I figured it'd be better to get rid of the complexity. For IP-level test, we can simply drive the sense pin on the IP and have no surprises. For chip-level test, software can do whatever we'd like, based on the selection for the peripheral's input mux.

Does that sound good to you? Or would you prefer the override feature be left in the IP?

Reply:
o that's a fair point. Now that you moved this to mio you get the free input 0 / 1 selection.
good call.